src/App.tsx

```diff
@@ -1,97 +1,94 @@
-              </div>
+    setQuery(inputQuery);
+    applyQuery(inputQuery);
+  };
 
-              <div
-                className="dropdown-item"
-                data-cy="suggestion-item"
-              >
-                <p className="has-text-link">Pieter Antone Haverbeke</p>
-              </div>
+  useEffect(() => {
+    if (!query) {
+      setSelectedPeople(null);
+    }
+  }, [query]);
 
-              <div
-                className="dropdown-item"
-                data-cy="suggestion-item"
-              >
-                <p className="has-text-danger">Elisabeth Haverbeke</p>
-              </div>
+  useEffect(() => {
+    setPageTitle(() => {
+      return selectedPeople
+        ? `${selectedPeople.name} (${selectedPeople.born} - ${selectedPeople.died})`
+        : 'No selected person';
+    });
+  }, [selectedPeople]);
 
-              <div
-                className="dropdown-item"
-                data-cy="suggestion-item"
-              >
-                <p className="has-text-link">Pieter de Decker</p>
-              </div>
+  const filteredPeople = useMemo(() => {
+    return peopleFromServer.filter((person) => person.name
+      .toLowerCase().includes(appliedQuery.trim().toLowerCase()));
+  }, [appliedQuery]);
 
-              <div
-                className="dropdown-item"
-                data-cy="suggestion-item"
-              >
-                <p className="has-text-danger">Petronella de Decker</p>
-              </div>
+  return (
+    <div className="container">
+      <main className="section">
+        <h1 className="title" data-cy="title">
+          {pageTitle}
 
-              <div
-                className="dropdown-item"
-                data-cy="suggestion-item"
-              >
-                <p className="has-text-danger">Elisabeth Hercke</p>
-              </div>
-            </div>
-          </div>
-        </div>
+        </h1>
 
         <div
-          className="
-            notification
-            is-danger
-            is-light
-            mt-3
-            is-align-self-flex-start
-          "
-          role="alert"
-          data-cy="no-suggestions-message"
+          className={classNames('dropdown', { 'is-active': !selectedPeople })}
         >
-          <p className="has-text-danger">No matching suggestions</p>
+          <InputPeople
+            query={query}
+            appliedQuery={appliedQuery}
+            setAppliedQuery={setAppliedQuery}
+            handleQueryChange={handleQueryChange}
+            setQuery={setQuery}
+            selectedPeople={selectedPeople}
+            setSelectedPeople={setSelectedPeople}
+            showList={showList}
+            setShowList={setShowList}
+            setPageTitle={setPageTitle}
+          />
+
+          <div className="dropdown-menu" role="menu">
+            {showList
+              && (filteredPeople.length ? (
+                <div className="dropdown-menu">
```

Comment:
If we show list you have two divs with the same class maybe it is better to use [react fragment](https://legacy.reactjs.org/docs/fragments.html) instead of div on 77 string?